client/reader/stats.js

```diff
@@ -110,6 +110,15 @@ export function recordTrackForPost( eventName, post = {}, additionalProps = {} )
 	}, additionalProps ) );
 }
 
+export function recordTrainTrackInteract( action, post = {} ) {
+	if ( post.railcar ) {
```

Comment:
would it be easier to just bake this into recordTrackForPost? We could just strip off the leading `calypso_reader` from the event name and report everything for free. Are there events reported through tracks we don't want to capture as interactions?

Tangent, but should we report the railcar to normal tracks events if it's available?